adhearsion-drb.gemspec

```diff
@@ -18,13 +18,12 @@ Gem::Specification.new do |s|
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths = ["lib"]
 
-  s.add_runtime_dependency "adhearsion", ["~> 2.0"]
-  s.add_runtime_dependency "i18n", ">= 0.5.0"
   s.add_runtime_dependency "jruby-openssl" if RUBY_PLATFORM == 'java'
+  s.add_runtime_dependency "adhearsion", "~> 3.0.0.rc1"
 
-  s.add_development_dependency "rspec", "~> 2.7.0"
+  s.add_development_dependency "rspec"
```

Comment:
I get the following error if I keep thor at "~> 0.14.0"

adhearsion-drb was resolved to 1.1.1, which depends on
      adhearsion (~> 3.0.0.rc1) was resolved to 3.0.0.rc1, which depends on
        thor (~> 0.18.0)

    guard-rspec was resolved to 4.7.3, which depends on
      guard (~> 2.1) was resolved to 2.14.0, which depends on
        thor (>= 0.18.1)